src/SonarScanner.MSBuild.PreProcessor/Protobuf/AnalysisCacheMsg.proto

```diff
@@ -0,0 +1,13 @@
+﻿/*
+  In order to update the generated file run
+  .\protoc.exe --proto_path=[sonar-scanner-msbuild-path]\src\SonarScanner.MSBuild.PreProcessor\Protobuf --csharp_out=[sonar-scanner-msbuild-path]\src\SonarScanner.MSBuild.PreProcessor\Protobuf [sonar-scanner-msbuild-path]\src\SonarScanner.MSBuild.PreProcessor\Protobuf\AnalysisCacheMsg.proto
+ */
+syntax = "proto3";
+
+option java_package = "org.sonar.scanner.protocol.internal";
```

Comment:
What's this for? We don't generate this as part of Java codebase
```suggestion
```

Reply:
This proto file is the contract between the scanner and the sonar-qube server. I've copied it as it is (except the c# namespace) and I prefer to not change it for 2 reasons:
- the data deserialization should work without surprises
- it will be easier to update if the server implementation changes

Reply:
I'd still remove it for the same reason you've added C# - we need to maintain this file properly.
On `sonar-dotnet` side, it will be `org.sonarsource.dotnet.protobuf` so we cannot copy it 1:1 and we need to maintain it properly. 
Having the scanner package raises maintainability question about where is it used on Java side of S4NET.
Having different value on sonar-dotnet makes it harder to maintain.